src/deps-index.ts

```diff
@@ -33,11 +35,12 @@ async function _getTransitiveDependencies(
     url: UrlString, entrypoints: UrlString[], analyzer: Analyzer):
     Promise<DependencyMapEntry> {
       const document = await analyzer.analyze(url);
+      const documentBase = document.parsedDocument.baseUrl;
```

Comment:
Given the very general meaning of `base`, I'd suggest going long with `documentBaseUrl`, or maybe just `baseUrl`.